packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -121,7 +137,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
 
   // Hide Tooltip
   @autobind
-  private _onTooltipMouseLeave(ev: any) {
+  private _onTooltipMouseLeave() {
```

Comment:
I'm using the same pattern for the 3 triggers: own focus, mouse or prop. If any is positive, tooltip is displayed; if any is negative, tooltip is hidden. This enables that when we display tooltip on focus (either own focus or setting the external prop), we can hide it with the mouse by entering and exiting the host element.